src/content/releases/_index.en.md

```diff
@@ -5,6 +5,20 @@ pre = "<b>6. </b>"
 weight = 30
 +++
 
+## v0.7.0 StatefulSet support for service discovery and benchmark tooling
+
+> This release mainly focused on adding support for StatefulSets in Lighthouse for service discovery and adding new `subctl`
+> commands to benchmark the network performance across clusters.
+
+* **Lighthouse** enhancements/changes:
+  * Added support for accessing individual pods in a StatefulSet using their host names.
```

Comment:
```suggestion
  * Added support for accessing individual Pods in a StatefulSet using their host names.
```

Reply:
I agree, Pods and Services are K8s objects and should be CamelCase.

https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects